ote/src/clj/ote/tasks/pre_notices.clj

```diff
@@ -91,51 +98,109 @@
+
+   [:table {:border "0", :width "80%", :cellpadding "0", :cellspacing "0"}
+    [:tbody
+     [:tr
+      [:td {:style "background:none; border-bottom: 1px solid #d7dfe3; height:1px; width:100%; margin:0px 0px 0px 0px;"} "&nbsp;"]]]]
+   [:br]])
+
+(defn- blue-button [link text]
+  [:table
+   [:tr
+    [:td.btn
+     [:a {:class "mcnButton"
+          :title text
+          :href link
+          :target "_blank"
+          :style "font-weight: normal;letter-spacing: normal;line-height: 100%;text-align: center;text-decoration: none;color: #FFFFFF;"}
+      text]]]])
+
+(defn notification-html [pre-notices detected-changes]
+  (println "pre-notices" (pr-str pre-notices))
+  [:html {:xmlns "http://www.w3.org/1999/xhtml"
+          :xmlns:v "urn:schemas-microsoft-com:vml"
+          :xmlns:o "urn:schemas-microsoft-com:office:office"}
+   [:head "<!-- NAME: 1 COLUMN - FULL WIDTH -->" "<!--
+                    [if gte mso 15]>\n<xml>\n<o:OfficeDocumentSettings>\n<o:AllowPNG/>\n<o:PixelsPerInch>96</o:PixelsPerInch>\n</o:OfficeDocumentSettings>\n</xml>\n<!
+                    [endif]-->"
+    [:meta {:charset "UTF-8"}]
+    [:meta {:http-equiv "X-UA-Compatible", :content "IE=edge"}]
+    [:meta {:name "viewport", :content "width=device-width, initial-scale=1"}]
+    [:title "NAP:ssa on uutta tietoa markkinaehtoisen liikenteen tulevista muutoksista."]
+    [:style {:type "text/css"}
+     ".headerText1 {font-family:roboto,helvetica neue,arial,sans-serif; font-size:2rem; font-weight:700;}
+     .headerText2 {font-family:roboto,helvetica neue,arial,sans-serif; font-size:1.5rem; font-weight:700;}
+     .whiteBackground {background-color:#FFFFFF}
+     .grayBackground {background-color:#EFEFEF}
+     .btn {background-color:#0066CC;padding:15px;}
+     .footer {font-size:0.75rem}
+     a.mcnButton{display:block;}
+     .even-row {background-color:#EFEFEF;}
+     .odd-row {background-color:#FFFFFF;}
+      .tg  {border-collapse:collapse;border-spacing:0;}
+      .tg td{font-family:roboto,helvetica neue,arial,sans-serif;font-size:14px;padding:10px 5px;overflow:hidden;word-break:normal;}
+      .tg th{font-family:roboto,helvetica neue,arial,sans-serif;font-size:16px;font-weight:700;padding:10px 5px;overflow:hidden;word-break:normal;}
+      .tg .tg-oe15{background-color:#ffffff;text-align:left;vertical-align:top}
+      .tg .tg-lusz{background-color:#656565;color:#ffffff;text-align:left;vertical-align:top}
+      .tg .tg-vnjh{text-decoration:underline;background-color:#ffffff;color:#0066cc;text-align:left;vertical-align:top}
+      .tg .tg-m03x{text-decoration:underline;background-color:#efefef;color:#0066cc;text-align:left;vertical-align:top}
+      .tg .tg-fkgn{background-color:#efefef;border-color:#efefef;text-align:left;vertical-align:top}"]]
+   [:body.grayBackground
+    [:center
+     [:div.whiteBackground {:width "80%" :style "padding-left: 20px; padding-right:20px;"}
+      [:img {:src (str (environment/base-url) "img/icons/NAP-logo-blue.png")
```

Comment:
I think the logo should be a link to the finap front page. It's often used best practice.

Reply:
Fixed